client/driver/rkt.go

```diff
@@ -731,6 +951,36 @@ func (d *RktDriver) Open(ctx *ExecContext, handleID string) (DriverHandle, error
 	return h, nil
 }
 
+func (d *RktDriver) fetchImage(img lastditch.RuntimeImage, fetchArgs []string) (string, error) {
```

Comment:
So this function could take globalArgs instead and do `fetchArgs := make([]String, 0, len(globalArgs) + 2 /* one for "fetch", one for image name */)`.